diff -Nur nomadnet-0.9.1.orig/nomadnet/__init__.py nomadnet-0.9.1/nomadnet/__init__.py
--- nomadnet-0.9.1.orig/nomadnet/__init__.py	2025-11-04 00:31:34.000000000 +0300
+++ nomadnet-0.9.1/nomadnet/__init__.py	2025-12-19 23:32:10.050388059 +0300
@@ -8,8 +8,10 @@
 from .ui import *
 
 
-modules = glob.glob(os.path.dirname(__file__)+"/*.py")
-__all__ = [ os.path.basename(f)[:-3] for f in modules if not f.endswith('__init__.py')]
+py_modules  = glob.glob(os.path.dirname(__file__)+"/*.py")
+pyc_modules = glob.glob(os.path.dirname(__file__)+"/*.pyc")
+modules     = py_modules+pyc_modules
+__all__ = list(set([os.path.basename(f).replace(".pyc", "").replace(".py", "") for f in modules if not (f.endswith("__init__.py") or f.endswith("__init__.pyc"))]))
 
 def panic():
     os._exit(255)
\ В конце файла нет новой строки
diff -Nur nomadnet-0.9.1.orig/nomadnet/ui/__init__.py nomadnet-0.9.1/nomadnet/ui/__init__.py
--- nomadnet-0.9.1.orig/nomadnet/ui/__init__.py	2025-11-04 00:31:34.000000000 +0300
+++ nomadnet-0.9.1/nomadnet/ui/__init__.py	2025-12-19 23:32:32.530181045 +0300
@@ -3,8 +3,10 @@
 import RNS
 import nomadnet
 
-modules = glob.glob(os.path.dirname(__file__)+"/*.py")
-__all__ = [ os.path.basename(f)[:-3] for f in modules if not f.endswith('__init__.py')]
+py_modules  = glob.glob(os.path.dirname(__file__)+"/*.py")
+pyc_modules = glob.glob(os.path.dirname(__file__)+"/*.pyc")
+modules     = py_modules+pyc_modules
+__all__ = list(set([os.path.basename(f).replace(".pyc", "").replace(".py", "") for f in modules if not (f.endswith("__init__.py") or f.endswith("__init__.pyc"))]))
 
 
 UI_NONE       = 0x00
diff -Nur nomadnet-0.9.1.orig/nomadnet/ui/textui/__init__.py nomadnet-0.9.1/nomadnet/ui/textui/__init__.py
--- nomadnet-0.9.1.orig/nomadnet/ui/textui/__init__.py	2025-11-04 00:31:34.000000000 +0300
+++ nomadnet-0.9.1/nomadnet/ui/textui/__init__.py	2025-12-19 23:32:42.146092491 +0300
@@ -1,5 +1,7 @@
 import os
 import glob
 
-modules = glob.glob(os.path.dirname(__file__)+"/*.py")
-__all__ = [ os.path.basename(f)[:-3] for f in modules if not f.endswith('__init__.py')]
+py_modules  = glob.glob(os.path.dirname(__file__)+"/*.py")
+pyc_modules = glob.glob(os.path.dirname(__file__)+"/*.pyc")
+modules     = py_modules+pyc_modules
+__all__ = list(set([os.path.basename(f).replace(".pyc", "").replace(".py", "") for f in modules if not (f.endswith("__init__.py") or f.endswith("__init__.pyc"))]))
diff -Nur nomadnet-0.9.1.orig/nomadnet/vendor/__init__.py nomadnet-0.9.1/nomadnet/vendor/__init__.py
--- nomadnet-0.9.1.orig/nomadnet/vendor/__init__.py	2025-11-04 00:31:34.000000000 +0300
+++ nomadnet-0.9.1/nomadnet/vendor/__init__.py	2025-12-19 23:32:52.058001214 +0300
@@ -1,5 +1,7 @@
 import os
 import glob
 
-modules = glob.glob(os.path.dirname(__file__)+"/*.py")
-__all__ = [ os.path.basename(f)[:-3] for f in modules if not f.endswith('__init__.py')]
+py_modules  = glob.glob(os.path.dirname(__file__)+"/*.py")
+pyc_modules = glob.glob(os.path.dirname(__file__)+"/*.pyc")
+modules     = py_modules+pyc_modules
+__all__ = list(set([os.path.basename(f).replace(".pyc", "").replace(".py", "") for f in modules if not (f.endswith("__init__.py") or f.endswith("__init__.pyc"))]))
